### PR TITLE
Add API versioning to backend

### PR DESCRIPTION
**Pull request by GPaoloni**
Jira: https://bugs.benetech.org/browse/CHI-308

This PR:
- Adds versioning (starting at v0), included in the url.
- Moves routes to a separate folder.
- Adds a new migration, which adds `accountSid` column to all of our tables. 

Details can be found in the Jira ticket linked above.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -60,6 +61,17 @@ function authorizationMiddleware(req, res, next) {
...
+ */
+const addAccountSid = (req, res, next) => {
+  req.accountSid = req.params.accountSid;
+  return next();
+};
+
+app.use('/v0/accounts/:accountSid', addAccountSid, apiV0);
+
 // run with node app.js and hit curl localhost:8080/contacts/
 app.get('/contacts', async (req, res) => {
```
Comment (andvirga) [inline app.js:76]: Your PR looks great! However I've seen that we're keeping the old routes in app.js file. Is this intentional? Should we delete them and start using the new routes?

Comment (GPaoloni) [reply app.js:76]: Yes, that's intentional, so we can change the endpoints being used in the frontend without breaking it. I'll remove them when we all feel confident that this is working fine.

**Review by andvirga — APPROVED** [review]

**Review by murilovmachado — APPROVED** [review]
Looks very good!!

**Comment by GPaoloni** [comment]
Deployed to staging.

**Pull request merged**